ui/src/main/js/view/widgets/dialog/index.js

```diff
@@ -59,7 +59,14 @@ exports.show = function(title, body, options) {
     bodyEl.append(body);
 
     options.modal = true;
-    var popover = popoverWidget.newPopover(title, dialog, undefined, options);
+    // Undefined for the onElement makes it in the window center
+    // WE NEED an onElement, but right now this fails
+    if (options.onElement) {
+        var popover = popoverWidget.newPopover(title, dialog, options.onElement, options);
+    } else {
+        var popover = popoverWidget.newPopover(title, dialog, undefined, options);
```

Comment:
Is this conditional required? I mean, when `options.onElement` is `undefined`, the resulting call is just the one on _else_ branch. Probably `var popover = popoverWidget.newPopover(title, dialog, options.onElement, options);` is enough (without the conditional logic).